getting-started/templates/systemlink-admin-values.yaml

```diff
@@ -1,40 +1,2 @@
-## Default values for the systemlink-admin Helm chart.
-## This is a YAML-formatted file.
-## Declare variables to be passed into your templates.
-
-global:
-  ## Defines secrets required if containers are hosted on a private image repository.
-  ##
-  imagePullSecrets: 
-  - &imagePullSecrets "niartifacts-secret"
-  ## Overrides the default image registry.
-  # <ATTENTION> - Use this override if mirroring the SystemLink container registry.
-  ##
-  imageRegistry: &imageRegistryRef "niedge01.jfrog.io/ni-docker"
-  ## Disable secret deployment if you want to manually manage secrets on the cluster.
-  ## WARNING: Changing this value from true to false during an upgrade will delete existing secrets.
-  # <ATTENTION> - Set to false if you do not want to manage secrets as part of the Helm installation.
-  ##
-  deploySecrets: true
-
-## Configuring Strimzi Kafka:
-## https://github.com/strimzi/strimzi-kafka-operator/tree/main/helm-charts/helm3/strimzi-kafka-operator#configuration
-##
-strimzi-kafka-operator:
-  ## Watch the whole Kubernetes cluster.
-  ##
-  watchAnyNamespace: true
-  podSecurityContext: 
-    runAsNonRoot: true
-  ## Uncomment to set Java heap size
-  ## Only required if the operator is crashing with
-  ## "Terminating due to java.lang.OutOfMemoryError: Java heap space"
-  #extraEnvs:
-  #  - name: JAVA_OPTS
-  #    value: "-Xms256m -Xmx256m"
-
-  ## The registry where kafka images are stored.
-  defaultImageRegistry: *imageRegistryRef
-  ## The secrets required to pull the images.
-  image:
-    imagePullSecrets: *imagePullSecrets
+## Templates are now versioned by release.
+## Refer to release-notes/<release version>/templates to getting started configuring your deployment
```

Comment:
No templates exist for older releases, but I guess that's not really a problem.

Reply:
Since we don't have a way to provide access to older releases (yet) this info is only helpful if you have a local copy of all our artifacts. Regardless, I think the risk of considerable negative effects on customers is negligible for now.

Reply:
We will need some way for customers to find a tagged version of a document without deep github knowledge.

Reply:
I'm going to update the Readme to point them towards the release. Also since this is docs for IT professionals I think we can have a reasonable expectation for familiarity with git/GitHub.